eo-runtime/src/main/java/org/eolang/PhDefault.java

```diff
@@ -156,7 +151,19 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        return this.toStringWith();
+        String result = String.format(
+            "%sv%d",
+            this.getClass().getCanonicalName(),
+            this.vertex
+        );
+        if (this.attrs.containsKey("Δ")) {
+            result= String.format(
```

Comment:
@mximp typo, missed space. Did you check this code with Checkstyle/PMD?

Reply:
@yegor256 You probably saw outdated changes.. I've made a commit several minutes later fixing Checkstyle issues.